extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -372,6 +382,118 @@ export class AmpAnalytics extends AMP.BaseElement {
     }
   }
 
+  /**
+   * Start resolving any macros that may exist in the linker configuration
+   * and register the callback with the navigation service. Since macro
+   * resolution is aynchronous the callback may be looking for these values
+   * before they are ready.
+   * @private
+   */
+  initLinker_() {
```

Comment:
let's separate this into a separate file `linker-manager.js`

Reply:
Done.